### DISCUSSION TITLE
Upgrade to RC1, Orchard Core CMS Recipes (beta3) failed

### DISCUSSION THREAD

**Discussion by brogits** [OrchardCMS/OrchardCore#9748] [Q&A]
I had a working Recipe in OrchardCore beta and  .net core 2.1. I started migrating it to OC RC1 and .net core 3.1 but found out that my Recipe is no longer available during setup.

I read from @davidhayden orchardcore.io that during migration of the beta site to rc1, he mentioned _a change to content part and content field settings ..._
(https://www.orchardcore.io/blog/upgrade-orchard-core-cms-website)

Any change you guys can point me how to get it sorted? Any documentation i can read and follow?

**Comment by ns8482e** [comment]
Can you share what kind of error are you getting?

**Reply by Skrypt** [reply]
He's right there's been changes in the structure of recipes. He might need to use the export feature to create his recipe file back.
